src/modules/components/rv-jump-to/directives/rv-jump-to-directive.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+(function (angular) {
+
+  angular
+    .module('rv.components.RVJumpToModule')
+    .directive('rvJumpTo', rvJumpTo);
+
+  function rvJumpTo() {
+
+    return {
+      restrict: 'AE',
+      scope: {},
+      bindToController: {
+        fieldNames: '=',
+        totalSections: '@'
+      },
+      controller: rvJumpToController,
+      controllerAs: 'vm',
+      templateUrl: 'modules/components/rv-center/templates/rv-jump-to-template.html'
+    };
+
+
+  }
+
+  rvJumpToController.$inject = ['$location', '$anchorScroll'];
+
+  function rvJumpToController($location, $anchorScroll) {
+    var vm = this; // jshint ignore:line
+
+    vm.gotoAnchor = function(field) {
+      if (angular.isDefined(totalSections)) {
```

Comment:
remove this condition check for totalSelections and rewrite in below ways

access all elements by name. e.g. document.getElementsByName(field)

This will give all the elements present in the page with this name and try to loop on this array. and scroll to that field.